programmers/p-y.js

```diff
@@ -0,0 +1,17 @@
+function solution(s){
+    var leng = s.length
```

Comment:
변수이름이 leng 좋은것 같지는 않습니다.
변수이름짓기 사이트에서 길이를 쳤더니 나오는 변수이름으로 대체했습니다.
변수이름을 지을때는 스네이크케이스와 카멜케이스가 있습니다.
구글링 해보시면 좋을거 같아요! 자바스크립트는 카멜케이스를 이용합니다.
```suggestion
    let linearContent = s.length
```